### PR TITLE
Added smart contract.

### PR DESCRIPTION
**Pull request by star-ranger01**
Created the CustomerAccount smart contract in Solidity.

### CODE REVIEW

File: CustomerAccount.sol
Comment (SkyRanger05) [inline CustomerAccount.sol:1]: This is excellent.

File: README.md
Comment (SkyRanger05) [inline README.md:1]: This is excellent.

**Review by SkyRanger05 — APPROVED** [review]
All is good.

File: README.md
Comment (star-ranger01) [reply README.md:1]: Ok.

**Comment by SkyRanger05** [comment]
Merge: reviewed and approved downstream star-ranger01.

**Pull request merged**